src/styles/main.scss

```
@@ -1,3 +1,73 @@
-body {
+$size: 100px;
+$wigth: $size * 7 + 17;
+
+* {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  line-height: 100px;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  margin: 200px auto;
+  width: $wigth;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  $days: 
+    ("mon": 2,
+    "tue": 103px,
+    "wed": 205px,
+    "thu": 307px,
+    "fri": 409px,
+    "sat": 511px,
+    "sun": 613px);
```

Здесь можно было бы добавить расчёты, чтобы сама переменная высчитывала значения, а не ты вручную!
```suggestion
  $days: 
    ("mon": 2,
    "tue": 103px,
    "wed": 205px,
    "thu": 307px,
    "fri": 409px,
    "sat": 511px,
    "sun": 613px);
```